### PR TITLE
feat: input component

### PR DESCRIPTION
**Pull request by Michael-Liendo**

### CODE REVIEW

File: src/lib/components/Input.svelte [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  export let name = '';
+  export let placeholder = '';
+  export let required = false;
+  export let id = '';
+  export let value = '';
+  export let autocomplete = '';
+</script>
+
+<div>
+  <label for="first-name" class="block text-sm font-medium text-gray-700">First name</label>
```
Comment (LeoBorai) [inline src/lib/components/Input.svelte]: Shouldn't this be:

```suggestion
  <label for="first-name" class="block text-sm font-medium text-gray-700">{label}</label>
```

With a corresponding `label` prop?

**Review by LeoBorai — CHANGES_REQUESTED** [review]
We are missing support for a `label` prop.

**Comment by LeoBorai** [comment]
@Michael-Liendo it would be great if we could have commits squashed into a single one before merging! Overall looks very good.

**Comment by Michael-Liendo** [comment]
> @Michael-Liendo it would be great if we could have commits squashed into a single one before merging! Overall looks very good.

sure!

**Review by LeoBorai — APPROVED** [review]
Wohoo! Thanks for contributing @Michael-Liendo!

**Pull request merged**